downstream/modules/platform/ref-controller-system-requirements.adoc

```diff
@@ -7,68 +7,39 @@ In the installer, four node types are provided as abstractions to help you desig
 
 Use the following recommendations for node sizing:
 
-[NOTE]
-====
-On control and hybrid nodes, allocate a minimum of 20 GB to `/var/lib/awx` for {ExecEnvShort} storage.
-====
-
 *Execution nodes* 
 
 Execution nodes run automation. Increase memory and CPU to increase capacity for running more forks.
 
 [NOTE]
 ====
-* The RAM and CPU resources stated might not be required for packages installed on an execution node but are the minimum recommended to handle the job load for a node to run an average number of jobs simultaneously.
+* The RAM and CPU resources stated are minimum recommendations to handle the job load for a node to run an average number of jobs simultaneously.
 
 * Recommended RAM and CPU node sizes are not supplied.
 The required RAM or CPU depends directly on the number of jobs you are running in that environment.
 
+* For capacity based on forks in your configuration, see link:https://docs.ansible.com/automation-controller/latest/html/userguide/jobs.html#automation-controller-capacity-determination-and-job-impact[Automation Controller Capacity Determination and Job Impact].
```

Comment:
Do we have this content in our own docset instead of linking to docs.ansible.com?